content_scripts/vimium.css

```diff
@@ -330,7 +330,7 @@ iframe.vimiumHUDFrame {
   right: 20px;
   margin: 0 0 0 -40%;
   border: none;
-  z-index: 2149999998; /* Two less than the reference value. */
+  z-index: 2139999998; /* Two less than the reference value. */
```

Comment:
Why this change?

Reply:
The 214... is an old typo, and make HUD hide (cover) some hint markers. While the related issue wants to see hints for elements which are at bottom-right corner and covered by HUD, so I think it's needed to show hint markers above HUD.